src/main/java/frc/lib/swerve/MAXSwerve.java

```diff
@@ -117,7 +119,7 @@ public void periodic() {
     m_chassisSpeed = m_kinematics.toChassisSpeeds(getModuleStates());
 
     m_odometry.update(
-        Rotation2d.fromDegrees(m_gyro.getAngle()),
+        Rotation2d.fromDegrees(-m_gyro.getAngle()),
```

Comment:
If the +/- sign here on the gyro angle may potentially change with hardware swaps consider adding it to Constants.

Reply:
Even better: you provide `getAngle()` lower down. If you use `getAngle()` instead of a direct call to the gyro you cut down on code repetition.